modules/renter/repair.go

```diff
@@ -336,6 +330,12 @@ func (r *Renter) managedScheduleChunkRepair(rs *repairState, chunkID chunkID, ch
 		missingPieces = missingPieces[:len(usefulWorkers)]
 	}
 
+	// Erasure code the pieces.
+	pieces, err := file.erasureCode.Encode(chunkData, missingPieces)
+	if err != nil {
+		return build.ExtendErr("unable to erasure code chunk data", err)
+	}
```

Comment:
the GC runs later, so it's kind of moot. I wish there was a way to Free() it immediately.